rtl/eclrtl/eclrtl.cpp

```diff
@@ -3762,6 +3766,45 @@ void rtlSeedRandom(unsigned value)
     random_->seed(value);
 }
 
+unsigned rtlPseudoRandomNumberBinomialDistribution(unsigned engine, double probablity, unsigned upper_bound)
+{
+    if (engine >= IPseudoRandomNumberGenerator::ePseudoRandomNumberEngine::NUMBER_OF_ENGINES)
+        return rtlRandom(); // Unknown engine, so just use default
```

Comment:
Surely picking a default engine would be better - otherwise it will not match the distribution.